src/main/java/org/beanpod/switchboard/dto/mapper/StreamMapper.java

```diff
@@ -15,5 +17,9 @@
 
   StreamDTO toDto(StreamEntity streamEntity);
 
+  List<StreamDTO> toDto(List<StreamEntity> streamEntityList);
```

Comment:
Can rename this  to `toDtos` since it's a list?

Reply:
Renamed it to `toDtoList`